package.json

```diff
@@ -7,7 +7,7 @@
     "@mui/styles": "^5.0.1",
     "@types/debounce-promise": "^3.1.3",
     "@veupathdb/components": "^0.8.3",
-    "@veupathdb/wdk-client": "^0.4.4",
+    "@veupathdb/wdk-client": "^0.4.5",
```

Comment:
Since the wdk-client is already merged, I'll comment here.

The centered text is not quite right.

I think what's needed is in FilterParam.css, the `right` below needs to be `center`
 
```
.filter-param .membership-filter .MesaComponent th:nth-child(3) .HeadingCell-Content-Label, .filter-param .membership-filter .MesaComponent th:nth-child(4) .HeadingCell-Content-Label {
	text-align: right;
}
```

Reply:
@bobular Thank you for finding the exceptions! 👍 yes you are right it is the place to change. I made a [PR at WDKClient](https://github.com/VEuPathDB/WDKClient/pull/159) after changing it. Here is a screenshot after the change:

![table-heading-centering](https://user-images.githubusercontent.com/12802305/137052100-8b9cfca8-567a-420a-a84c-b0a1ebc5b3f5.png)